### PR TITLE
[MIG] account_invoice_commission to 11.0

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

File: account_invoice_commission/README.rst [hunk 1]
```diff
@@ -48,23 +56,22 @@ To configure this module, you need to:
...
+To use this module, you need to:
 
-Known issues / Roadmap
-======================
+#. Go to ...
 
-* ...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: http://runbot.adhoc.com.ar/
```
Comment (zaoral) [inline account_invoice_commission/README.rst:68]:
```
+   :target: http://runbot.adhoc.com.ar/
```
Need to be https: http page does not work.

File: account_invoice_commission/models/account_invoice.py [hunk 2]
```diff
@@ -53,21 +53,18 @@ class AccountInvoice(models.Model):
         compute='_compute_partner_user',
```
Comment (zaoral) [inline account_invoice_commission/models/account_invoice.py:52]: remove line

```
 # from odoo.exceptions import ValidationError
```
is really not used

File: account_invoice_commission/models/account_invoice_line.py [hunk 3]
```diff
@@ -18,14 +17,12 @@ class AccountInvoiceLine(models.Model):
         currency_field='company_currency_id',
     )
 
-    @api.multi
-    @api.depends('commission_amount')
     def _compute_commission_amount(self):
```
Comment (zaoral) [inline account_invoice_commission/models/account_invoice_line.py:20]: @nicomacr Why you removed the api.depends from this method?

File: account_invoice_commission/views/account_invoice_view.xml [hunk 4]
```diff
@@ -37,7 +37,7 @@
             </field>
             <notebook>
                 <page string="Commission">
-                    <field name="commissioned_invoice_ids" context="{'commissioned_partner_id': partner_id, 'tree_view_ref':'account.invoice_tree', 'form_view_ref':'account.invoice_form', 'search_default_invoices': 1, 'search_default_refunds': 1, 'search_default_not_commissioned': True, 'search_default_paid': 1, 'search_default_user_id': partner_user_id}"/>
+                    <field name="commissioned_invoice_ids"  context="{'commissioned_partner_id': partner_id, 'tree_view_ref':'account_invoice_commission.invoice_tree', 'form_view_ref':'account.invoice_form', 'search_default_invoices': 1, 'search_default_refunds': 1, 'search_default_not_commissioned': True, 'search_default_paid': 1, 'search_default_user_id': partner_user_id}"/>
```
Comment (zaoral) [inline account_invoice_commission/views/account_invoice_view.xml]: avoid extra space after before context option added when updating in this line

File: account_invoice_commission/models/account_invoice_line.py [hunk 5]
```diff
@@ -18,14 +17,12 @@ class AccountInvoiceLine(models.Model):
...
         commissioned_partner_id = self._context.get('commissioned_partner_id')
         if commissioned_partner_id:
             today = fields.Date.context_today(self)
             rules = self.env['account.commission.rule']
-            _logger.info('Computing commission amount')
+            _logger.info('Computing commission amount line')
```
Comment (zaoral) [inline account_invoice_commission/models/account_invoice_line.py:25]: Should we use _() for translation?

**Review by zaoral — CHANGES_REQUESTED** [review]
@nicomacr 
Here I leave you some comments, please review when you can.

Best Regards

File: account_invoice_commission/README.rst (hunk 1 above)
Comment (zaoral) [reply account_invoice_commission/README.rst:68]: sorry, my mistake. it's the opposite. http is the one that works.
Do not take into account this comment.

File: account_invoice_commission/models/account_invoice_line.py (hunk 5 above)
Comment (jjscarafia) [reply account_invoice_commission/models/account_invoice_line.py:25]: for log messages it seams better to me to keep them in english (without translation)

Comment (zaoral) [reply account_invoice_commission/models/account_invoice_line.py:25]: > for log messages it seams better to me to keep them in english (without translation)

ok. good to know

**Pull request merged**